.github/workflows/Tests.yaml

```diff
@@ -170,8 +170,14 @@ jobs:
           - build_type: Debug
             EXTRA_CXX_FLAGS: "-DBLAZE_USE_VECTORIZATION=0"
           - compiler: clang-8
-            build_type: Release
+            build_type: Debug
+            # When building with ASAN we also need python bindings to be
+            # disabled because otherwise we get link errors:
+            # ImportError: bin/python/spectre/Informer/_PyInformer.so:
+            # undefined symbol: __asan_option_detect_stack_use_after_return
+            ASAN: ON
             BUILD_PYTHON_BINDINGS: OFF
```

Comment:
The clang-8-Debug-ASAN build is now the only one without Pybindings. Is that intentional, or should we (also) disable Pybindings in another build that has no ASAN and/or is a Release build?

Reply:
Yes, that's intentional. I could also disable the bindings in release, but probably more pressing would be testing for python 2 and python 3. I can add a disable on a GCC9 release so we cover also compiling without bindings on GCC9 and on release. Thoughts?

Reply:
keeping the configuration as-is is also fine with me, we'll have to reconsider how we distribute python configs over the builds in #1782 anyway.

Reply:
Alright, sounds good!